tests/acceptance/31_testall_selftest/skipped.cf

```diff
@@ -0,0 +1,18 @@
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+bundle agent test
+{
+  meta:
+    "test_skip_needs_work" string => "any";
+}
+
+bundle agent check
+{
+  reports:
+    "$(this.promise_filename) Pass";
```

Comment:
Shouldn't this say `FAIL` so that it only passes if skipped?

Reply:
Sure. That'd be fine. It is skipped and passes regardless because the test is not run at all.